modules.py

```diff
@@ -43,6 +43,14 @@ def reply(bot, message, intent, entities):
         result = random.choice(['Heads', 'Tails'])
         bot.send_photo(message.chat.id, photo=coin_images[result],
                        reply_to_message_id=message.message_id)
+    elif intent[:9] == 'translate':
+        word = message.text[9:]
```

Comment:
This would work, but it's not advisable. Ideally, you'd define an entity containing the text to be translated.